codebuild/cd/generic-unix-build.sh

```diff
@@ -43,3 +37,8 @@ JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64 mvn -B package -DskipTests -Dshared
 
 aws s3 cp --recursive --include "*.so" target/cmake-build/lib s3://aws-crt-java-pipeline/${GIT_TAG}/lib
 aws s3 cp target/ s3://aws-crt-java-pipeline/${GIT_TAG}/jar/ --recursive --exclude "*" --include "aws-crt*.jar"
+
+if [[ $AWS_CRT_TARGET != linux-armv8 ]]; then
+    # the FIPS subset contains FIPS build for linux-armv8 and linux-x64, and NON-FIPS build for rest of the linux platforms.
```

Comment:
your comments says "linux-armv8 and linux-x64" but the if-statement is just for "linux-armv8"
Is this a bug, or am I confused?